jmx-metrics/src/integrationTest/resources/hadoop/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM bmedora/hadoop:2.9-base
+
+COPY yarn-site.xml /hadoop/etc/hadoop/yarn-site.xml
```

Comment:
It looks like you should be able to use copyFileToContainer instead of a Dockerfile. We definitely want to avoid building a docker image during test execution if at all possible

Reply:
Thanks for the suggestion. I was able to implement it following the example in Cassandra and it definitely speeds up the testing time.